src/main/java/at/hannibal2/skyhanni/features/misc/customscoreboard/ScoreboardPattern.kt

```diff
@@ -0,0 +1,9 @@
+package at.hannibal2.skyhanni.features.misc.customscoreboard
+
+import at.hannibal2.skyhanni.utils.repopatterns.RepoPattern
+
+object ScoreboardPattern {
```

Comment:
As long as this class only holds one pattern, it surely can be moved into CustomScoreboardUtils

Reply:
Im currently counting 44 on my end, will push soon:tm: